doc/man7/EVP_KDF-HKDF.pod

```diff
@@ -135,16 +135,16 @@ RFC 5869
 =head1 SEE ALSO
 
 L<EVP_KDF>,
```

Comment:
`L<EVP_KDF(3)>`

(needs to be fixed in multiple places)

Reply:
Fixed now.

There are some `B<EVP_KDF>` left but they are discussion the structure pointers.